upb/msg_internal.h

```diff
@@ -65,6 +65,7 @@ struct upb_msglayout {
   uint16_t size;
   uint16_t field_count;
   bool extendable;
+  uint8_t dense_below;
```

Comment:
dense_below maxes at 256, but I don't see any test for that in your build above

Reply:
Good catch, I checked this in the C++ codegen but not in the dynamic table building. Done.